src/app/core/basic-datatypes/configurable-enum/configure-enum-popup/configure-enum-popup.component.spec.ts

```diff
@@ -84,6 +85,48 @@ describe("ConfigureEnumPopupComponent", () => {
     expect(confirmationSpy).toHaveBeenCalledWith(
       "Delete option",
       `Are you sure that you want to delete the option "${male.label}"?`,
+      jasmine.any(Array),
     );
   });
+
+  it("should set newOptionInput with pasted multiline text and prevent default paste", () => {
+    const pastedText = "Option A\nOption B\nOption C";
+
+    const clipboardData = {
+      getData: jasmine.createSpy("getData").and.returnValue(pastedText),
+    };
+
+    const preventDefaultSpy = jasmine.createSpy("preventDefault");
+
+    const fakeEvent = {
+      clipboardData: clipboardData,
+      preventDefault: preventDefaultSpy,
+    } as unknown as ClipboardEvent;
+
+    component.onPasteNewOption(fakeEvent);
+
+    expect(clipboardData.getData).toHaveBeenCalledWith("text");
+    expect(component.newOptionInput).toBe("Option A\nOption B\nOption C");
+    expect(preventDefaultSpy).toHaveBeenCalled();
+  });
+
+  it("should not prevent default if pasted text is a single line", () => {
+    const pastedText = "Single Option";
+
+    const clipboardData = {
+      getData: jasmine.createSpy("getData").and.returnValue(pastedText),
+    };
+
+    const preventDefaultSpy = jasmine.createSpy("preventDefault");
+
+    const fakeEvent = {
+      clipboardData: clipboardData,
+      preventDefault: preventDefaultSpy,
+    } as unknown as ClipboardEvent;
+
+    component.onPasteNewOption(fakeEvent);
+
+    expect(component.newOptionInput).toBeUndefined(); // because splitByLine returns only 1 line
```

Comment:
should the pasted "Single Option" be added somewhere now? not in newOptionInput? Can you add an expectation for that single item being added here?